PowerShell/JumpCloud Commands Gallery/Windows Commands/Windows - Set ReleaseChannel In JumpCloud Password Manager.md

```diff
@@ -0,0 +1,35 @@
+#### Name
+
+Windows - Set JumpCloud Password Manager's Release Channel | v1.0 JCCG
+
+#### commandType
+
+windows
+
+#### Command
+
+```
+# Set $RELEASE_CHANNEL to beta OR dogfood OR public depending on your desired release channel
+$RELEASE_CHANNEL = "public"
```

Comment:
We can't expect admins to do the right thing, it would probably be a good idea to add a check to ensure that `$RELEASE_CHANNEL` is one of 'public','dogfood','beta'